rts/Sim/Units/UnitDefHandler.cpp

```diff
@@ -225,20 +225,21 @@ void CUnitDefHandler::SetNoCost(bool value)
 
 void CUnitDefHandler::SanitizeUnitDefs()
 {
-	for (size_t i = 1; i < unitDefsVector.size(); i++) {
-		UnitDef& ud = unitDefsVector[i];
-		
-		//Sanitize unitDef.corpse (wreckName)
-		if (ud.wreckName != "") {
-			const FeatureDef* wreckFeatureDef = featureDefHandler->GetFeatureDef(ud.wreckName); 
-			if (wreckFeatureDef == nullptr) {
-				ud.wreckName = "";
-			}
-		}
+	for (auto &ud : unitDefsVector) {
+		// Sanitize unitDef.wreckName
+		if (ud.wreckName == "")
+			continue;
```

Comment:
Ah, but now that the function handles more than just wrecks it can't be an early return anymore.

Reply:
Yes, I noticed that as soon as I pushed :)